contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -382,7 +385,7 @@ def _execute(self, function_search_string):
         functions_dataview.get_item_at(0, 0).click_input('right')
 
         self.find_context_menu_item('Enable frame track(s)').click_input()
-        wait_for_condition(lambda: '✓ F' in functions_dataview.get_item_at(0, 0).texts()[0])
+        wait_for_condition(lambda: selected_function_icon + ' F' in functions_dataview.get_item_at(0, 0).texts()[0])
```

Comment:
Is the `'F'` only used once? Otherwise maybe define a variable for that too, it's a bit strange to have a variable for one and not the other.